ckanext/showcase/tests/test_plugin.py

```diff
@@ -1,4 +1,4 @@
-from routes import url_for
+from ckan.lib.helpers import url_for
```

Comment:
For extra points next time use `toolkit.url_for` :ok_hand:

Reply:
will do, thanks for the hint and for merging :)